client/src/app/users/add-user.component.html

```diff
@@ -23,8 +23,7 @@
 
           <mat-form-field>
             <mat-label>Age</mat-label>
-            <input matInput #input id="ageField" type="number" placeholder="Age"
-                  formControlName="age" required>
+            <input matInput #input type = "number" id="ageField" placeholder="Age"                  formControlName="age" required>
```

Comment:
Do we want the spaces in the `type` assignment?

Reply:
Oops, nope, thanks! 😂